ddtrace/context.py

```diff
@@ -26,7 +32,8 @@
         _MetaDictType,  # _meta
         _MetricDictType,  # _metrics
     ]
-
+# regex matches ",", ";", ":", and characters outside the ASCII range 0x20 to 0x7E
+TRACESTATE_INVALID_CHARS_REGEX = r",|;|:|[^\x20-\x7E]+"
```

Comment:
This should be internal

```suggestion
_TRACESTATE_INVALID_CHARS_REGEX = r",|;|:|[^\x20-\x7E]+"
```